File: server/router/ActivitiesManagement.go
package router

import (
	"gin-vue-admin/api/v1"
	"gin-vue-admin/middleware"
	"github.com/gin-gonic/gin"
)

func InitActivitiesManagementRouter(Router *gin.RouterGroup) {
	ActivitiesManagementRouter := Router.Group("acm").Use(middleware.OperationRecord())
	{
		ActivitiesManagementRouter.POST("createActivitiesManagement", v1.CreateActivitiesManagement)   // 新建ActivitiesManagement
		ActivitiesManagementRouter.DELETE("deleteActivitiesManagement", v1.DeleteActivitiesManagement) // 删除ActivitiesManagement
		ActivitiesManagementRouter.DELETE("deleteActivitiesManagementByIds", v1.DeleteActivitiesManagementByIds) // 批量删除ActivitiesManagement
		ActivitiesManagementRouter.PUT("updateActivitiesManagement", v1.UpdateActivitiesManagement)    // 更新ActivitiesManagement
		ActivitiesManagementRouter.GET("findActivitiesManagement", v1.FindActivitiesManagement)        // 根据ID获取ActivitiesManagement
		ActivitiesManagementRouter.GET("getActivitiesManagementList", v1.GetActivitiesManagementList)  // 获取ActivitiesManagement列表
		ActivitiesManagementRouter.GET("getAttendedActivitiesInfoList", v1.GetAttendedActivitiesInfoList)  // 获取ActivitiesManagement列表
		ActivitiesManagementRouter.PUT("updateAttendedActivitiesInfoList", v1.UpdateAttendedActivitiesInfoList)  // 获取ActivitiesManagement列表
		ActivitiesManagementRouter.GET("getAttendant", v1.GetAttendant)
		ActivitiesManagementRouter.PUT("updateAttendant", v1.UpdateAttendant)
		ActivitiesManagementRouter.GET("getAttendantList", v1.GetAttendantList)
	}
}

File: web/src/api/ActivitiesManagement.js
import service from '@/utils/request'

// @Tags ActivitiesManagement
// @Summary 创建ActivitiesManagement
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body model.ActivitiesManagement true "创建ActivitiesManagement"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"获取成功"}"
// @Router /acm/createActivitiesManagement [post]
export const createActivitiesManagement = (data) => {
     return service({
         url: "/acm/createActivitiesManagement",
         method: 'post',
         data
     })
 }


// @Tags ActivitiesManagement
// @Summary 删除ActivitiesManagement
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body model.ActivitiesManagement true "删除ActivitiesManagement"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"删除成功"}"
// @Router /acm/deleteActivitiesManagement [delete]
 export const deleteActivitiesManagement = (data) => {
     return service({
         url: "/acm/deleteActivitiesManagement",
         method: 'delete',
         data
     })
 }

// @Tags ActivitiesManagement
// @Summary 删除ActivitiesManagement
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body request.IdsReq true "批量删除ActivitiesManagement"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"删除成功"}"
// @Router /acm/deleteActivitiesManagement [delete]
 export const deleteActivitiesManagementByIds = (data) => {
     return service({
         url: "/acm/deleteActivitiesManagementByIds",
         method: 'delete',
         data
     })
 }

// @Tags ActivitiesManagement
// @Summary 更新ActivitiesManagement
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body model.ActivitiesManagement true "更新ActivitiesManagement"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"更新成功"}"
// @Router /acm/updateActivitiesManagement [put]
 export const updateActivitiesManagement = (data) => {
     return service({
         url: "/acm/updateActivitiesManagement",
         method: 'put',
         data
     })
 }


// @Tags ActivitiesManagement
// @Summary 用id查询ActivitiesManagement
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body model.ActivitiesManagement true "用id查询ActivitiesManagement"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"查询成功"}"
// @Router /acm/findActivitiesManagement [get]
 export const findActivitiesManagement = (params) => {
     return service({
         url: "/acm/findActivitiesManagement",
         method: 'get',
         params
     })
 }


// @Tags ActivitiesManagement
// @Summary 分页获取ActivitiesManagement列表
// @Security ApiKeyAuth
// @accept application/json
// @Produce application/json
// @Param data body request.PageInfo true "分页获取ActivitiesManagement列表"
// @Success 200 {string} string "{"success":true,"data":{},"msg":"获取成功"}"
// @Router /acm/getActivitiesManagementList [get]
 export const getActivitiesManagementList = (params) => {
     return service({
         url: "/acm/getActivitiesManagementList",
         method: 'get',
         params
     })
 }




export const getAttendedActivitiesInfoList = (params) => {
    return service({
        url: "/acm/getAttendedActivitiesInfoList",
        method: 'get',
        params
    })
}
